src/map/script.cpp

```diff
@@ -25736,6 +25736,20 @@ BUILDIN_FUNC( openstylist ){
 #endif
 }
 
+BUILDIN_FUNC(getitempos) {
+	TBL_PC *sd;
+	if (script_rid2sd(sd)){
```

Comment:
Please invert the logic and return SCRIPT_CMD_FAILURE here.
If script_rid2sd fails it reports this to the script engine and actually cancels the script execution.
So nothing will be pushed to the script engine in this case.